namer/namer.cc

```diff
@@ -2020,6 +2020,20 @@ class TreeSymbolizer {
             }
         }
 
+        // Prevent a crash by validating the arguments before we do any symbol lookups or manipulations
+        if (send->hasPosArgs() || send->hasKwArgs() || send->block() != nullptr) {
+            if (send->numPosArgs() > 1) {
+                if (auto e = ctx.beginError(send->loc, core::errors::Namer::InvalidTypeDefinition)) {
+                    e.setHeader("Too many args in type definition");
+                }
+                auto send = ast::MK::Send0Block(asgn.loc, ast::MK::T(asgn.loc), core::Names::typeAlias(),
```

Comment:
The fact that this didn’t work before (it was using Send1) indicates we might also want to have something in the test that uses the untyped constant (e.g. use `X` in the test below in a signature of a method and use T.reveal_type to check that it gets treated as T.untyped)